### PR TITLE
Import data task

### PR DESCRIPTION
**Pull request by Layoric**
Create an admin task to allow importing of large folder of documents to enhance and store.

Also create controller to use while we are still prototyping to clear data via rest call.

### CODE REVIEW

**Comment by Layoric** [comment]
@scelton Could have a quick once over, mostly admin tasks that won't persist in the code base for long but to help us debug dev.

File: src/test/java/au/gov/ga/hydroid/admintasks/FileImportTestIT.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+               else
+                  System.out.print("File '" + filePath.toFile().getName() + "' succeeded.");
+            }
+         } catch (Exception e) {
+            e.printStackTrace();
+         }
+      });
+   }
+
+   private Integer postFile(String uri,File file) throws IOException {
```
Comment (scelton) [inline src/test/java/au/gov/ga/hydroid/admintasks/FileImportTestIT.java]: I noticed you implemented a postFile method in RestClientImpl, were you planning to use/test it here?

**Comment by scelton** [comment]
Other than the postFile question/comment, all looks good to me!

File: src/test/java/au/gov/ga/hydroid/admintasks/FileImportTestIT.java (hunk 1 above)
Comment (Layoric) [inline src/test/java/au/gov/ga/hydroid/admintasks/FileImportTestIT.java]: Good catch! Redundant due to `postFile` impl in test itself. Had trouble with RestEasy any ways. Removed.

**Pull request merged**